### ISSUE TITLE
come on guys, how agar infinity works, there gotta be something to do

### ISSUE THREAD

**Issue by sejo153** [davidmann4/ogar-feeder-bot#590]
even myb edit agar infinity to have 10 bots

**Comment by polarbear2g** [comment]
I only get 2 bots, pretty lame.

**Comment by ZfsrGhS953** [comment]
In a few weeks, I will start working on bots for SCRAMBLED Ogar and MultiOgar servers :). Check it out at ZfsrGhS953/agario-client

**Comment by fayizan** [comment]
@sejo153  how did u edit it....

**Comment by ZfsrGhS953** [comment]
Currently it's not working. It will be started in a few weeks.

**Comment by fayizan** [comment]
@ZfsrGhS953 please don't make minions for Ogar instead make it for agar.io

**Comment by ZfsrGhS953** [comment]
Yes, but I need to make minions for Ogar first to understand how to make them for agar.io.

**Comment by NuclearC** [comment]
@ZfsrGhS953 i already made bots for multiogar, add me on skype :nuclear.agar i can help you

**Comment by ZfsrGhS953** [comment]
You mean you made bots for scramble level 2 and 3? I'm asking because agario.city has scramble level 1. BTW I'm trying to decode protocol 9, check it out here: https://github.com/ZfsrGhS953/agario-client It still doesn't work, but I edited some code by observing how MultiOgar sends packets to clients with protocol 6,7,8 (and 9, that one was added yesterday)

**Comment by NuclearC** [comment]
no it's lvl 1 :P anyway official agario uses something like scramble lvl 1 because it's not sending fake border packets like MultiOgar's lvl 2.
EDIT: everything is actually implemented, only i need to learn some maths and it'll work on scramble lvl 2 :)
EDIT2: it can connect to one of agario servers (151.80.98.51:1523) and get world size:
width: 14142.135623730952
height: 14142.135623730952
scrambleX: 484.95489429498230
scrambleY: -3960.2037243526775

**Comment by ZfsrGhS953** [comment]
Did you decrypt protocol 9?

**Comment by NuclearC** [comment]
no i'm sending in order:
[!] Sent: fe 09 00 00 00
[!] Sent: ff  00 35 fa dd
server response:
[!] Recv: ff 2b 00 00 00 f0 1c 40 ae 06 d6 76 8c b2 c1 c0 8b 03 c5 95 b1 8d c1 c0 a3 93 90 ca 09 d9 b3 40 ea 99 b2 8c bf 22 b4 40 08 00 00 00 38 2e 30 2e 32 00 : world size
[!] Recv: f1 4b 09 5e a7 : IDK
[!] Recv: 3b 0b 92 4e 7a 04 95 4c 7d 0f 9d 4a 7f 09 91 4b 7e 08 9c 49 : world update or lb
EDIT: I think f1 is owned nodes id

**Comment by ImHITMan** [comment]
Reason why agarinfinity works is because the bots are based on a "Peer 2 Peer" System which basically means, It's hard to ban the IP's used for the bots because they keep switching
